apps/web/src/modules/job/appliedJob/pages/styled.ts

```diff
@@ -0,0 +1,81 @@
+import { styled } from '@mui/material'
+
+export const SearchContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  ${(props) => props.theme.breakpoints.down('md')} {
+    display: none;
+  }
+`
+
+export const JobContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 3rem;
+  width: 45vw;
+  gap: 1rem;
+  padding-bottom: 3rem;
+  ${(props) => props.theme.breakpoints.down('lg')} {
+    width: 60vw;
+  }
+  ${(props) => props.theme.breakpoints.down('md')} {
+    width: 95vw;
+  }
+  ${(props) => props.theme.breakpoints.up('md')} {
+    display: flex;
+  }
```

Comment:
Not neccesary ? I think it's always `display: flex` (`> md` => flex, otherwise => flex)

Reply:
Job container display is change to none when filterPage is open (in mobile), I try to make it appear normally when change screen size from mobile to desktop without closing filterPage.